src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -651,6 +659,78 @@ handle_call(exchange, _From, #state{store=Store}=State) ->
     Pid = spawn(fun() -> do(fold, [Store, Function, []]) end),
     {reply, {ok, Pid}, State};
 
+handle_call(delta_gc, _From, #state{store=Store}=State) ->
+    Function =
+        fun({Id, #dv{delta_map=DeltaMap0, delta_ack_map=AckMap0,
+                     delta_counter=Counter}=_Object},
+            Acc0) ->
+                case orddict:is_empty(DeltaMap0) of
+                    true ->
+                        Acc0;
+                    false ->
+                        %% Remove disconnected or slow node() entries from the AckMap.
+                        %% disconnected or slow: seen the previous GC & no change
+                        RemovedAckMap0 =
+                            orddict:filter(fun(_Node, {Ack, GCed}) ->
+                                                   (GCed == false) orelse (Ack == Counter)
+                                           end, AckMap0),
+                        case orddict:size(RemovedAckMap0) of
+                            0 ->
+                                Acc0;
+                            _ ->
+                                %% Mark remained entries as seen this GC.
+                                RemovedAckMap =
+                                    orddict:fold(
+                                      fun(Node, {Ack, _GCed}, RemovedAckMap1) ->
+                                              orddict:store(Node, {Ack, true},
+                                                            RemovedAckMap1)
+                                      end, orddict:new(), RemovedAckMap0),
+                                %% Collect garbage deltas.
+                                MinAck =
+                                    lists:min([Ack || {_Node, {Ack, _GCed}} <- RemovedAckMap]),
+                                %% size() should be bigger than 0.
+                                MinAck1 =
+                                    case orddict:size(DeltaMap0) of
+                                        1 ->
+                                            MinAck;
+                                        _ ->
+                                            Counters = orddict:fetch_keys(DeltaMap0),
+                                            NotCollectable =
+                                                (MinAck > lists:nth(1, Counters)) andalso
+                                                    (MinAck < lists:nth(2, Counters)),
```

Comment:
This comparison is needed when the `MinAck` is in the accumulator slot (then, the `MinAck` will be adjusted). If it is equal to boundaries, the rest of code for garbage collection will handle it properly.